COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
@jrafanie @Fryguy The RestartForceExitStatus was released in systemd 215 (http://lists.freedesktop.org/archives/systemd-devel/2014-July/020903.html).  We are currently running 208 on the appliance.  I'm going to try to install the newer one and see if I can use SuccessExitStatus=123 and RestartForceExitStatus=123 to fix all the issues.